packages/office-addin-lint/src/lint.ts

```diff
@@ -26,8 +31,14 @@ export function getLintCheckCommand(files: string): string {
 }
 
 export function performLintCheck(files: string) {
-  const command = getLintCheckCommand(files);
-  execCommand(command);
+  try {
+    const command = getLintCheckCommand(files);
+    execCommand(command);
+    usageDataHelper.sendUsageDataSuccessEvent("check");
```

Comment:
`check` is very generic. `performLintCheck` -- same as the function -- seems more appropriate.

Also, put the name into a const variable which is used for both success and error reporting. That pattern ensures that the same name is used.

`const usageDataName = "performLintCheck";`

usageDataHelper.sendUsageDataSuccessEvent(usageDataName);
or
usageDataHelper.sendUsageDataException(usageDataName, err);